package.json

```diff
@@ -33,7 +33,7 @@
     "lint": "tslint --project . && eslint test/",
     "test": "nyc mocha",
     "report-coverage": "nyc report --reporter=json && codecov -f coverage/*.json",
-    "integration": "integration-loader && integration all",
+    "integration": "integration-loader && integration all -- --exit",
```

Comment:
What is `--exit`? I don't see it in https://github.com/interledgerjs/five-bells-integration-test/blob/master/src/bin/integration.

Reply:
As there are unterminated listeners lingering in tests/tested components (STREAM and possible Connector), Mocha 4 doesnt automatically exit the tests (https://github.com/mochajs/mocha/issues/3044)

Adding the `-- --exit` passes the exit flag to mocha and allows the tests to exit correctly after finished running

Reply:
Unterminated listeners seems like a bug that should be fixed, not sidestepped, but this is an alright stopgap.